packages/metascraper-clearbit-logo/index.js

```diff
@@ -5,18 +5,21 @@ const got = require('got')
 
 const DEFAULTS = {
   size: '128',
-  format: 'png'
+  format: 'png',
+  greyscale: false
 }
 
 const ENDPOINT = 'https://logo.clearbit.com'
 
 module.exports = opts => {
   opts = Object.assign({}, DEFAULTS, opts)
-  const { size, format } = opts
+  const { size, format, greyscale } = opts
 
   const clearbitLogo = async ({ url }) => {
     const { hostname } = new URL(url)
-    const logoUrl = `${ENDPOINT}/${hostname}?size=${size}&format=${format}`
+    const logoUrl = `${ENDPOINT}/${hostname}?size=${size}&format=${format}${
```

Comment:
looks like `greyscale=false` doesn't work and `greyscale=true` is the same than `greyscale`

Reply:
Yup, any value for `greyscale` will cause the filter - it's a Clearbit thing